test/commons/color/get-background-color.js

```diff
@@ -7,19 +7,59 @@ describe('color.getBackgroundColor', function () {
   var origBodyBg;
   var origHtmlBg;
 
+  /**
+   * Assert that two Colors are equal.
+   * @param {axe.commons.color.Color[]} actual
+   * @param {axe.commons.color.Color[]} expected
+   */
+  function assertColorsEqual(actual, expected) {
+    assert.equal(actual.red, expected.red, 'red');
+    assert.equal(actual.green, expected.green, 'green');
+    assert.equal(actual.blue, expected.blue, 'blue');
+    assert.equal(actual.alpha, expected.alpha, 'alpha');
+  }
+
+  /**
+   * Assert that two Colors are close-to-equal.
+   * @param {axe.commons.color.Color[]} actual
+   * @param {axe.commons.color.Color[]} expected
+   * @param {number} threshold How much each RGB value may differ by
+   * @param {number} alphaThreshold How much the alpha channel may differ by
+   */
+  function assertColorsClose(
+    actual,
+    expected,
+    threshold = 0.5,
+    alphaThreshold = 0.1
+  ) {
+    assert.closeTo(actual.red, expected.red, threshold, 'red');
+    assert.closeTo(actual.green, expected.green, threshold, 'green');
+    assert.closeTo(actual.blue, expected.blue, threshold, 'blue');
+    assert.closeTo(actual.alpha, expected.alpha, alphaThreshold, 'alpha');
+  }
+
   before(function () {
     origBodyBg = document.body.style.background;
     origHtmlBg = document.documentElement.style.background;
   });
 
-  afterEach(function () {
-    document.body.style.background = origBodyBg;
-    document.documentElement.style.background = origHtmlBg;
+  beforeEach(function () {
+    // This normalizes the default mocha behavior of setting a different background
+    // based on prefers-color-scheme settings.
+    document.body.style.background = '#fff';
+    document.documentElement.style.background = 'unset';
+  });
 
+  afterEach(function () {
     axe.commons.color.incompleteData.clear();
     axe._tree = undefined;
   });
 
+  after(function () {
+    document.body.style.background = origBodyBg;
```

Comment:
This shouldn't be necessary anymore either as the [global `afterEach` function](https://github.com/dequelabs/axe-core/blob/develop/test/testutils.js#L601) from testutils removes the `style` attribute from body

Reply:
Updated the global afterEach to also clear `<html>` styles similarly to `<body>` ones